asreview/feature_extraction/sbert.py

```diff
@@ -18,7 +18,20 @@ def _check_st():
 
 
 class SBERT(BaseFeatureExtraction):
-    """Sentence BERT class for feature extraction."""
+    """Sentence BERT class for feature extraction.
```

Comment:
I think we might consider removing this one. Last time I tried to install it, it failed because of old build dependencies...